docs/user-guide.md

```diff
@@ -712,6 +715,28 @@ the root of the repository.
 *Note:* Currently only Strings type values are supported as parameters to
 CloudFormation templates when deploying via AWS CodePipeline.
 
+**CloudFormation Parameters in a Multi-Organization ADF Setup** 
+The CloudFormation Parameter generation feature is fully compatible with 
+the [Multi-Organization ADF Setup](./multi-organization-guide.md) approach. 
+Let's assume that we have a three AWS Org setup with a dev, int and prod 
+AWS Organization. This implies that the SSM param `/adf/org/stage` will have 
+one of the following three values: `[dev, int, prod]`; depending on the AWS 
+Organization you are in. Let's further assume that your application in scope 
+requires AWS Organization specific parameters.In that case, the `params` should 
+have the following content: 
+
+```txt
+params
+└───global_dev.yml
+└───global_int.yml
+└───global_prod.yml
+└───global.yml
+```
+
+When the application gets deployed, ADF will choose the right parameter file 
+based on the value of the SSM prameter "/adf/org/stage".
+
```

Comment:
```suggestion
Where it will prefer the AWS Organization specific configuration
`global_${org_stage}` over the `global` parameters in case they both
match the same parameter or tag.
```